src/patterns/components/card/react/code/base.hbs

```diff
@@ -0,0 +1,5 @@
+<SprkCard idString="card-1">
+  <div className="sprk-o-Stack__item sprk-c-Card__content sprk-o-Stack sprk-o-Stack--large">
+    Base Card Content
```

Comment:
We want this code to match whats in the doc site so when users copy it, they get what they expect. Need to remove "Base Card Content".